### DISCUSSION TITLE
Can't upload document to verify academic status

### DISCUSSION THREAD

**Discussion by rustamismagilov** [community/community#133303] [GitHub Education]
### Hi everyone,

Hi, for some reason, when I'm uploading photo of my student ID on the 
[https://education.github.com/discount_requests/XXXXXXXX/additional_information](https://education.github.com/discount_requests/XXXXXXXX/additional_information) page, it's just reloading without any changes.
I've tried it several times, but didn't achieved anything.
Any help appreciated, thanks!

**Comment by ghost** [comment]
Check File Type and Size or
Disable Browser Extensions: Browser extensions or add-ons could potentially interfere with the upload process. Try disabling them temporarily and then attempt the upload again or Contact GitHub Support.

**Comment by capitanfeeder** [comment]
I had the same issue. 
When I was uploading a photo taken with my phone, it was giving me that error. 
What I did was take a photo with the webcam, and it let me continue.

**Reply by Maru6622** [reply]
I tried to use my phone as a webcam with an app, but still have the same issue. I have really bad web-cam on both PC and phone, also i could not switch from web to front cam when applying with phone. What to do?

**Comment by evandrojr** [comment]
Same problem here! That is happening for at least one week. It is a shame they don´t fix it!

**Comment by ravilovrm** [comment]
Do the documents need to be translated into English or will the original work as well?

**Comment by capitanfeeder** [comment]
Yes, I uploaded them in Spanish but they were rejected because they need to
be translated into English.

**Reply by ravilovrm** [reply]
That is, you need to show the original student ID in the photo and put a printout with the translation next to it? Is this suitable? I already sent
![Uploading image_2024-07-24_18-48-44.png…]()

**Reply by ravilovrm** [reply]
https://imgur.com/a/6HoH5qM
